lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -3,6 +3,7 @@
 """
 from rest_framework.reverse import reverse
 
+from xmodule.modulestore.mongo.base import BLOCK_TYPES_WITH_CHILDREN
```

Comment:
It would be nice for that list to not be just part of the mongo modulestore implementation. Not for this PR, but perhaps we should generalize this to making `load_classes` take a filtering parameter.